tests/unit/utils/convert-ms-word-html-test.ts

```diff
@@ -0,0 +1,1240 @@
+       </body>
+    </html>
+    `;
+
+    const inputParser = new HTMLInputParser({});
+    const actualHtml = convertMsWordHtml(htmlContent, inputParser);
+
+    assert.strictEqual(expectedHtml, actualHtml);
+  });
+
+  test('It should display formatted list as HTML', function (assert) {
+    const inputParser = new HTMLInputParser({});
+    const expectedHtml = oneLineTrim`
+    <ul>
+     <li><span>Feature</span><span>1</span></li>
+     <li><span>Feature</span><span>2</span></li>
+    </ul>
+    `;
+    const htmlContent = oneLineTrim`
+      <html xmlns:o="urn:schemas-microsoft-com:office:office" xmlns:w="urn:schemas-microsoft-com:office:word" xmlns:m="http://schemas.microsoft.com/office/2004/12/omml" xmlns="http://www.w3.org/TR/REC-html40">
+         <head>
+            <meta http-equiv=Content-Type content="text/html; charset=utf-8">
+            <meta name=ProgId content=Word.Document>
+            <meta name=Generator content="Microsoft Word 15">
+            <meta name=Originator content="Microsoft Word 15">
+         </head>
+         <body lang=en-AL style='tab-interval:36.0pt;word-wrap:break-word'>
+            <!--StartFragment-->
+            <p class=MsoListParagraphCxSpFirst style='text-indent:-18.0pt;mso-list:l0 level1 lfo1'>
+               <![if !supportLists]><span lang=EN-US style='font-size:20.0pt;font-family:Symbol;mso-fareast-font-family: Symbol;mso-bidi-font-family:Symbol;mso-ansi-language:EN-US'><span style='mso-list:Ignore'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp; </span></span></span><![endif]><span lang=EN-US style='font-size:14.0pt; mso-ansi-language:EN-US'>Feature</span><span lang=EN-US style='font-size:14.0pt'> </span><span lang=EN-US style='font-size:14.0pt;mso-ansi-language:EN-US'>1</span>
+               <span lang=EN-US style='font-size:20.0pt;mso-ansi-language:EN-US'>
+                  <o:p></o:p>
+               </span>
+            </p>
+            <p class=MsoListParagraphCxSpLast style='text-indent:-18.0pt;mso-list:l0 level1 lfo1'>
+               <![if !supportLists]><span lang=EN-US style='font-size:20.0pt;font-family:Symbol;mso-fareast-font-family: Symbol;mso-bidi-font-family:Symbol;mso-ansi-language:EN-US'><span style='mso-list:Ignore'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp; </span></span></span><![endif]><span lang=EN-US style='font-size:14.0pt; mso-ansi-language:EN-US'>Feature</span><span lang=EN-US style='font-size:14.0pt'> </span><span lang=EN-US style='font-size:14.0pt;mso-ansi-language:EN-US'>2</span>
+               <span lang=EN-US style='font-size:20.0pt;mso-ansi-language:EN-US'>
+                  <o:p></o:p>
+               </span>
+            </p>
+            <!--EndFragment-->
+         </body>
+      </html>
+    `;
+
+    const actualHtml = convertMsWordHtml(htmlContent, inputParser);
+    assert.strictEqual(expectedHtml, actualHtml);
```

Comment:
qunit expects the first argument to be actual and the second to be expected
it makes a difference in the test reporting when it fails, so it's fairly important to get it right